cpp/benchmarks/io/text/multibyte_split.cpp

```diff
@@ -78,6 +80,23 @@ static cudf::string_scalar create_random_input(int32_t num_chars,
   return cudf::string_scalar(std::move(*chars_buffer));
 }
 
+static void write_bgzip_file(cudf::host_span<char const> host_data, std::ostream& stream)
```

Comment:
Using `stream` name is misleading as people may misunderstand it as cuda stream.

Reply:
@upsj This comment was resolved but it doesn't appear that there was a reply nor was the comment addressed. I think this is a good idea to consider avoiding the name `stream`.